src/components/common/StyledSelect/components/CustomIndicatorsContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import { IconButton, useColorModeValue } from "@chakra-ui/react"
+import { transparentize } from "@chakra-ui/theme-tools"
+import { GroupBase, IndicatorsContainerProps, components } from "chakra-react-select"
+import { motion } from "framer-motion"
+import { Check, Copy } from "phosphor-react"
+import { useState } from "react"
+import parseFromObject from "utils/parseFromObject"
+import { StyledSelectProps } from "../StyledSelect"
+
+const MotionIconButton = motion(IconButton)
+
+const CustomIndicatorsContainer = ({
+  children,
+  value,
+  pathToCopy,
+  ...rest
+}: {
+  value: StyledSelectProps["value"]
+  pathToCopy: string
+} & IndicatorsContainerProps<unknown, boolean, GroupBase<unknown>>) => {
+  const [copied, setCopied] = useState(false)
+  const successIconColor = useColorModeValue("green.800", "green.200")
+
+  const handleCopyToClipboard = async () => {
+    try {
+      await navigator.clipboard.writeText(parseFromObject(value, pathToCopy))
```

Comment:
Is it intentional that you haven't used the `useClipboard` hook here?

Reply:
Wow I missed this, thanks!